compiler/rustc_monomorphize/src/partitioning/mod.rs

```diff
@@ -149,7 +150,13 @@ fn get_partitioner<'tcx>(tcx: TyCtxt<'tcx>) -> Box<dyn Partitioner<'tcx>> {
 
     match strategy {
         "default" => Box::new(default::DefaultPartitioning),
-        _ => tcx.sess.fatal("unknown partitioning strategy"),
+        _ => {
+            let error_message = "unknown partitioning strategy".to_string();
+            // FIXME: Should be emit_fatal
```

Comment:
Why can't you add an `emit_fatal`?

Reply:
I don't want to duplicate https://github.com/rust-lang/rust/pull/100694